app/helpers/application_helper.rb

```diff
@@ -415,4 +415,8 @@ def edit_inline(object, property, options = {})
       h(value)
     end
   end
+
+  def table_css_classes
```

Comment:
Its all good :).
Ill leave my fix like that for now, but i do think your right, ill open a PR for it if none exist.